src/App.tsx

```diff
@@ -96,29 +125,40 @@ const App: React.FC = () => {
     // Tree structure
     try {
       const res = await fetch(
-        `https://api.github.com/repos/${owner}/${repo}/commits/master`
+        GITHUB_API_URL_PREFIX +
+          `${owner}/${repo}` +
+          GITHUB_API_COMMITS_ON_MASTER_SUFFIX
       );
       const resJSON = await res.json();
-      const treeSHA = resJSON["commit"]["tree"]["sha"];
+      const treeSHA =
+        resJSON[GithubData.COMMIT][GithubData.TREE][GithubData.SHA];
       const treeRes = await fetch(
-        `https://api.github.com/repos/${owner}/${repo}/git/trees/${treeSHA}?recursive=true`
+        GITHUB_API_URL_PREFIX +
+          `${owner}/${repo}` +
+          GITHUB_API_TREES_SUFFIX +
+          `/${treeSHA}` +
+          WITH_RECURSIVE_PARAMETER
       );
       const treeJSON = await treeRes.json();
 
-      const allItems = treeJSON["tree"].length;
+      const numberOfItems = treeJSON[GithubData.TREE].length;
 
-      for (let index = 0; index < allItems; index += 1) {
-        const item = treeJSON["tree"][index];
-        if (item.type == "blob") {
+      for (let index = 0; index < numberOfItems; index += 1) {
+        const item = treeJSON[GithubData.TREE][index];
+        if (item.type == IS_FILE) {
           const SHA = item.sha;
+          const path = item.path;
           const blobs = await fetch(
-            `https://api.github.com/repos/${owner}/${repo}/git/blobs/${SHA}`
+            GITHUB_API_URL_PREFIX +
+              `${owner}/${repo}` +
```

Comment:
If you are using a template literal there is no need to use + signs

Reply:
like ` ${GITHUB_API_URL_PREFIX}${owner}/${repo}` ?